src/resources/js/commerce/gateway/stripe/checkout.js

```diff
@@ -456,6 +454,15 @@ tribe.tickets.commerce.gateway.stripe.checkout = {};
 		}
 	};
 
+	/**
+	 * Get purchaser form data.
+	 *
+	 * @since TBD
+	 *
+	 * @return {Object}
+	 */
+	obj.getPuchaserInfo = () => tribe.tickets.commerce.getPurchaserData( $( tribe.tickets.commerce.selectors.purchaserFormContainer ) );
```

Comment:
Why are we naming it PurchaserInfo here and PurchaserData on the other file? We should pass `$container` into this method, and fall back into the selector in case the container is not passed.

Reply:
`getPurchaserInfo` is named differently to make it separate from the base `tribe.tickets.commerce.getPurchaserData`  method otherwise if both had same name then it might be confusing.

Here in the stripe JS we don't have any generic container that we can use that is why I have just used the default selector directly and wrapped the whole thing in a smaller named method for simplicity.

If we ever need to alter the container then we can do that in the future within the `getPurchaserInfo` method but I don't see any advantage of having the `$cotainer` param here again as this will defeat the whole purpose of this function to simplify the call.

Reply:
Sounds good on the second item for the container. The first one I still dont get why we would name somethinng diff if that is nothing other then this checkout "module" extending the one from the base module, so makes sense to name it the same.